addons/website/static/src/scss/website.wysiwyg.scss

```diff
@@ -33,6 +33,31 @@ html[data-edit_translations="1"] {
     }
 }
 
+.o_snippet_override_invisible {
+    display: block !important;
+    opacity: 70%;
+    position: relative;
+
+    &::before {
+        position: absolute;
+        width: -webkit-fill-available;
+        width: -moz-available;
```

Comment:
Is not fit-content supported? Although, I am not sure of the need... maybe add some comments?

Reply:
The `fit-content` value could be used when the icon was an FA character.
Now the icon is an SVG mask - and to avoid noise the rendered unwanted (but mandatory) text is reduced to 0px,